lib/handle-pull-request-change.js

```diff
@@ -2,7 +2,18 @@ module.exports = handlePullRequestChange
 
 async function handlePullRequestChange (robot, context) {
   const title = context.payload.pull_request.title
-  const isWip = /\b(wip|do not merge)\b/i.test(title)
+  var isWip = /\b(wip|do not merge)\b/i.test(title)
+
+  const commits = await context.github.pullRequests.getCommits({
+    owner: context.payload.repository.owner.login,
+    repo: context.payload.repository.name,
+    number: context.payload.pull_request.number
+  })
+
+  if (commits.data.some(element => { return /\b(wip)\b/i.test(element.commit.message) })) {
```

Comment:
I think it makes sense to re-use `isWip` from above, if you extract it as a function and pass string argument to it

Reply:
Thanks for the review @HeeL ! I pushed a commit that follows your suggestion